proto/internalapi/central/hello.proto

```diff
@@ -24,6 +24,9 @@ message SensorHello {
 
   // Policy version sensor understands. If unset, central will try to guess it.
   string policy_version = 4;
+
+  // Reconnect signfies that connecting sensor is reconnecting after connection was interrupted
+  bool reconnect = 6;
```

Comment:
I would vote for using an enum here, as there may be soon other cases that we want to cover. Examples:
- Sensor restarted after error
- Sensor restarted after benign connection termination (see our customer cases)
- Sensor is reconnecting without restart

Reply:
I mean this only for the proto message as keeping the backwards-compat here has much more value. The atomic bool in Sensor code can stay in my opinion.

Reply:
Makes sense. I'll make this change